pubsub/pubsub.go

```diff
@@ -76,8 +76,8 @@
 //
 // Since Message.Nack never makes sense for some providers (for example, for
 // at-most-once providers, the provider can't redeliver the message), Nack will
-// panic if called for some providers. You can call Message.CanNack to see if
-// it is available.
+// panic if called for some providers. You can call Subscription.CanNack to see
```

Comment:
Thanks for catching this! Actually, this sentence should be deleted entirely. `Subscription.CanNack` is on the driver interface type (`driver.Subscription`, not `pubsub.Subscription`), so it is not exposed to end users. We are considering exposing it to end users via `Message.Nackable` (currently commented out below).

So, do you mind just deleting the sentence starting with "You can call"?

Reply:
Sure. Done now.